lib/pages/new_group/new_group.dart

```diff
@@ -112,6 +113,12 @@ class NewGroupController extends State<NewGroup> {
     final client = Matrix.of(context).client;
 
     try {
+      if (nameController.text.isEmpty &&
```

Comment:
```suggestion
      if (nameController.text.trim().isEmpty &&
```

Small suggestion: We should trim the entered text so that we don't allow just whitespaces as a name

Reply:
Makes sense! I've updated this